inc/Engine/CDN/Subscriber.php

```diff
@@ -346,4 +348,20 @@ private function is_allowed() {
 	private function is_cdn_enabled() {
 		return (bool) $this->options->get( 'cdn', 0 );
 	}
+
+	/**
+	 * Check if CDN can insert resource hints into head.
+	 *
+	 * @return bool
+	 */
+	private function can_insert_resource_hints() {
```

Comment:
Let's try to keep up with code standards :sweat_smile: 
```suggestion
	private function can_insert_resource_hints(): bool {
```